src/Presentation/AmountFormatter.php

```diff
@@ -14,8 +14,8 @@ class AmountFormatter {
 	private $locale;
 
 	private $localeData = [
-		'de_DE' => [2, ',', '' ],
-		'en_US' => [2, '.', '' ]
+		'de_DE' => [2, ',', ''],
+		'en_US' => [2, '.', '']
```

Comment:
It's not really consistent imo: When accessing array elements there are no spaces, only when defining them in a single line.